### DISCUSSION TITLE
3.6 哈希表

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#115] [Comments]
# 3.6 哈希表

1. 哈希表简介 哈希表（Hash Table）：也叫做散列表。是根据关键码值（Key Value）直接进行访问的数据结构。 哈希表通过「键 $key$」和「映射函数 $Hash(key)$」计算出对应的「值 $value$」，把关键码值映射到表中一个位置来访问记录，以加快查找的速度。这个映射函数叫做「哈希函数（散列函数）」，存放记录的数组叫做「哈希表...

[https://algo.itcharge.cn/03_stack_queue_hash_table/03_06_hash_table/](https://algo.itcharge.cn/03_stack_queue_hash_table/03_06_hash_table/)

**Comment by Sycml** [comment]
哈希表简介里向哈希表中插入一个关键码值其中的" Hash(key)=0138//100=0"应该是0138//1000吧

**Comment by Yuki-J1** [comment]
配合 https://algo.hufeifei.cn/OpenHash.html 食用味更好
